pkg/machinehealthcheck/machinehealthcheck.go

```diff
@@ -0,0 +1,32 @@
+package machinehealthcheck
```

Comment:
Would this package make more sense under `clusterapi`?

Reply:
sure, I can move it. i saw a similar reconcile for aws iam auth at the pkg level so replicated it.